packages/core-mobile/app/store/settings/appearance/types.ts

```diff
@@ -6,10 +8,12 @@ export enum Appearance {
 
 export const DEFAULT_APPEARANCE = Appearance.System
 
-export const initialState = {
-  selected: DEFAULT_APPEARANCE
+export const initialState: AppearanceState = {
+  selected: DEFAULT_APPEARANCE,
+  colorScheme: null
```

Comment:
should we default this to light? I think colorScheme being dark or light should be enough.